becquerel/tools/isotope_qty.py

```diff
@@ -467,18 +505,22 @@ def _mul_div(self, other, div=False):
             factor = 1 / float(other)
         else:
             factor = float(other)
+        key = next(iter(self._ref_quantities))
         return IsotopeQuantity(
-            copy.deepcopy(self.isotope), date=self.ref_date,
-            atoms=self.ref_atoms * factor)
+            copy.deepcopy(self.isotope),
+            **{"date": self.ref_date, key: self._ref_quantities[key] * factor}
+        )
 
     def __eq__(self, other):
         """Equality operation"""
 
         if not isinstance(other, IsotopeQuantity):
             return False
         else:
+            # TODO: Do we need to compare all quantities?
```

Comment:
I think probably we test one, and `atoms` seems like a good one, but I am open.

Reply:
Yeah I agree. Just checked, the defaults in `numpy.isclose` are `rtol=1e-05, atol=1e-08`. The absolute tolerance should never be hit here as we handle order of 10^20 atoms, but the relative tolerance is kind of small, we might want to bump this to maybe 1e-08 or so?